scripts/Connector/Connector_Hirose/conn_hirose_df63_tht_top.py

```diff
@@ -94,7 +96,7 @@ def generate_one_footprint(pins, configuration):
     footprint_name = footprint_name.replace("__",'_')
 
     kicad_mod = Footprint(footprint_name)
-    kicad_mod.setDescription("Molex {:s}, {:s}, {:d} Pins per row ({:s}), generated with kicad-footprint-generator".format(series_long, mpn, pins_per_row, datasheet))
+    kicad_mod.setDescription("Hirose {:s}, {:s}, {:d} Pins per row ({:s}), generated with kicad-footprint-generator".format(series_long, mpn, pins_per_row, datasheet))
```

Comment:
`pins_per_row` can't be printed as a range and leads to a double space. I think `pins` is correct?

Reply:
`pins` is correct, but i don't see the double space you're mentioning in the generated footprints.
i changed it, because it's correct to use `pins`, but the generated footprints differ only in the edit time